Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -188,6 +205,48 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       return _indexColumnRenderer;
     }
 
+    private void RenderValidationRow (
+        BocListRenderingContext renderingContext,
+        BocListRowRenderingContext rowRenderingContext,
+        BocListDataRowRenderEventArgs rowRenderEventArgs,
+        IReadOnlyCollection<BocListValidationFailureWithLocationInformation> validationFailuresWithLocationInfo,
+        int rowIndex)
+    {
+      var cssClassValidationRow = GetCssClassValidationRow(rowRenderingContext.IsSelected, rowRenderEventArgs);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, cssClassValidationRow);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Tr);
+
+      var columnRenderers = renderingContext.ColumnRenderers;
+
+      var validationColumnRenderer = columnRenderers.First(r => r.ColumnDefinition is BocValidationErrorIndicatorColumnDefinition);
```

Comment:
Is it guaranteed that there is always a BocValidationErrorIndicatorColumnDefinition when this code is reached? should we still be a bit more expressive and use FirstOrDefault and then add an `Assertion.IsNotNull(..., "BocValidationErrorIndicatorColumnDefinition is not proivded.")` to give better diagnostics / document our assumtions?